jinishakheni/BrainBash-Frontend
# 24 create event editing  create form

Added Edit Event button when the host of the event visualized event detail page.
Also, added navigation to login when user is not logged in and clicks join button of the event.

closes #24 

Test:
- Edit event. Verify validation (validation of categories and skills has been commented at this point).
- Edit even. Save and observe changes on event detail page
